ansible/roles/debops.gitlab/tasks/main.yml

```diff
@@ -45,6 +45,14 @@
     state: 'present'
   when: gitlab_support_filesystem_acl|bool
 
+- name: Allow GitLab user to the ~/.ssh directory
+  file:
+    path: '{{ gitlab_home + "/.ssh" }}'
+    owner: '{{ gitlab_user }}'
+    group: '{{ gitlab_group }}'
+    mode: 0644
+    recurse: true
```

Comment:
I see this issue crop up multiple times now, I wonder when `~/.ssh/` is created by user other than `git` during GitLab deployment...

I'm not sure if `recurse: true` is needed here, are the files inside of this directory not owned by `git` user?

The mode is completely bonkers, this has to be `0700`, otherwise OpenSSH server will refuse to work. Also, you should quote the mode, otherwise it will be interpreted as octal number.